src/Collapse.js

```diff
@@ -62,29 +60,39 @@ class Collapse extends Component {
     return this.props.horizontal ? node.scrollWidth : node.scrollHeight;
   }
 
-  onEntering(node, isAppearing) {
+  getNode() {
+    const nodeRef = this.props.innerRef || this.nodeRef;
+    return nodeRef.current;
```

Comment:
From the above recommendation, we can then do
```suggestion
  return this.nodeRef.current;
```
For here and in the render method